engine/src/main/java/org/camunda/bpm/engine/impl/jobexecutor/JobExecutorLogger.java

```diff
@@ -213,6 +213,10 @@ public void infoJobExecutorDoesNotHandleBatchJobs(ProcessEngineConfigurationImpl
         config.getBatchJobPriority());
   }
 
+  public void debugFailedJobListenerSkipped(String jobId) {
+    logWarn("031", "Failed job listener skipped for job {} because it's been already re-acquired", jobId);
```

Comment:
Should we really log this on warn? If this happens often (since the user has configured a low expiration time), it potentially creates many logs that need to be stored on disk. This might impact the performance of process execution. I propose logging it in DEBUG to enable it in a test environment when required.